app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -59,12 +65,16 @@ protected void onCreate(Bundle savedInstanceState) {
 
         btnLogin.setOnClickListener(v -> {
 
-            String email = etEmail.getText().toString();
+            String email = etEmail.getEditText().getText().toString();
             String password = etPassword.getText().toString();
 
             presenter.login(email, password);
         });
 
+        if(presenter.getEmailList() != null)
```

Comment:
Don't do this. Create a presenter start method, and it will call `attachEmails` on the view, if it has emails, or else, it won't